### PR TITLE
Complete FontFile ref and update example

### PR DESCRIPTION
**Pull request by CorentinBrule**
I complete the ref items according to [TextServer ref](https://docs.godotengine.org/en/latest/classes/class_textserver.html) and update example with new properties path

### CODE REVIEW

File: doc/classes/FontFile.xml [hunk 1]
```diff
@@ -17,13 +17,13 @@
 		[codeblocks]
 		[gdscript]
 		var f = load("res://BarlowCondensed-Bold.ttf")
-		$"Label".set("custom_fonts/font", f)
-		$"Label".set("custom_fonts/font_size", 64)
+		$"Label".set("theme_override_fonts/font", f)
+		$"Label".set("theme_override_font_sizes/font_size", 64)
```
Comment (YuriSizov) [inline doc/classes/FontFile.xml:21]:
```suggestion
		$Label.add_theme_font_override("font", f)
		$Label.add_theme_font_size_override("font_size", 64)
```

File: doc/classes/FontFile.xml [hunk 2]
```diff
@@ -17,13 +17,13 @@
...
 		[/gdscript]
 		[csharp]
 		var f = ResourceLoader.Load&lt;FontFile&gt;("res://BarlowCondensed-Bold.ttf");
-		GetNode("Label").Set("custom_fonts/font", f);
-		GetNode("Label").Set("custom_font_sizes/font_size", 64);
+		GetNode("Label").Set("theme_override_fonts/font", f);
+		GetNode("Label").Set("theme_override_font_sizes/font_size", 64);
```
Comment (YuriSizov) [inline doc/classes/FontFile.xml:26]:
```suggestion
		GetNode("Label").AddThemeFontOverride("font", f);
		GetNode("Label").AddThemeFontSizeOverride("font_size", 64);
```

**Review by YuriSizov — CHANGES_REQUESTED** [review]
These code samples weren't proper to begin with. Your change is correct, but we should change them further.

**Comment by CorentinBrule** [comment]
I had not come across the doc of the Control object and this way of overriding theme, I have always used the editor's property path 😳. I found the same outdated example in the [FontVariation class ref](https://github.com/godotengine/godot/blob/master/doc/classes/FontVariation.xml)

**Comment by YuriSizov** [comment]
> I found the same outdated example in the [FontVariation class ref](https://github.com/godotengine/godot/blob/master/doc/classes/FontVariation.xml)

Feel free to fix it the same way :)

**Comment by CorentinBrule** [comment]
> > I found the same outdated example in the [FontVariation class ref](https://github.com/godotengine/godot/blob/master/doc/classes/FontVariation.xml)
> 
> Feel free to fix it the same way in this PR :)

in this PR ? Give me a few minutes to understand how to do this :)

**Comment by YuriSizov** [comment]
Yes, you need to update it anyway to apply my suggestions. You can create a second commit to add more changes, and then you need to squash them and force-push the branch. See [our documentation](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase) if you need help with that.

**Comment by CorentinBrule** [comment]
Sorry I'm not very good with Git, by the time I understood how to follow your expected workflow (only 1 commit per PR) I accepted your suggestions from the github interface, which created a commit that I couldn't figure out. BUT, the new modification was (i hope) well done.

**Review by YuriSizov — APPROVED** [review]
Yes, you're good now :)

**Comment by akien-mga** [comment]
Thanks! And congrats for your first merged Godot contribution :tada:

**Pull request merged**